tests/e2e/fixtures/layers.fixture.json

```diff
@@ -0,0 +1,108 @@
+{
+  "ch.bfe.windenergie-geschwindigkeit_h50": {

```

Comment:
I guess the layer selection here is not random, it might be helpful to add a comment why you chose these particular layers (cause they are probably typical representatives of a certain kind of layer?)

Reply:
that was to minimize changes in my tests, but I could surely embrace the "I make my layer myself" in the test, and rename them accordingly. It will make more sense

Reply:
so now we have really fake layers with fake IDs for test, this way there's no confusion